cmd/containerd-shim-runc-v2/process/init.go

```diff
@@ -141,9 +141,16 @@ func (p *Init) Create(ctx context.Context, r *CreateConfig) error {
 		opts.ConsoleSocket = socket
 	}
 
-	if err := p.runtime.Create(ctx, r.ID, r.Bundle, opts); err != nil {
+	createCtx, createCancel := context.WithTimeout(context.Background(), 30*time.Second)
+	defer createCancel()
+	if err := p.runtime.Create(createCtx, r.ID, r.Bundle, opts); err != nil {
 		return p.runtimeError(err, "OCI runtime create failed")
 	}
+
+	if ctx.Err() != nil {
```

Comment:
why is this checking `ctx.Err()`? I assume you meant to check `createCtx.Err()`?